engine/execution/computation/computer/computer.go

```diff
@@ -306,7 +314,7 @@ func (e *blockComputer) executeTransaction(
 	collectionIndex int,
 	txIndex uint32,
 	res *execution.ComputationResult,
-) error {
+) (error, txerror error) {
```

Comment:
```suggestion
) (executionErr error, txErr error) {
```
pretty sure go will want you to name both return values